Core/StitchCoreAdminClient/Sources/StitchCoreAdminClient/StitchAdminAuth.swift

```diff
@@ -22,7 +22,7 @@ public final class StitchAdminAuth: CoreStitchAuth<StitchAdminUser> {
         return info
     }
 
-    public final override func onAuthEvent() {
-        // do nothing
+    public final override func dispatchAuthEvent(_ authEvent: AuthRebindEvent) {
+
     }
```

Comment:
I personally like when these empty functions are only one line like 

 public final override func dispatchAuthEvent(_ authEvent: AuthRebindEvent) { } 

I feel like we use that pattern elsewhere (esp in JS sdk) but if that is not the pattern we want dont worry / let me know.

Reply:
I agree that it's a little cleaner. We did the other way because in Java the linter doesn't like it on a single line, but for Swift I'll try to start using the single-line approach.